AppXManifest.xml

```diff
@@ -17,7 +17,7 @@ Licensed under the MIT License.
   <Properties>
     <DisplayName>Accessibility Insights for Web</DisplayName>
     <PublisherDisplayName>Microsoft Corporation</PublisherDisplayName>
-    <Logo>extension\icons\brand\blue-on-white\brand-blue-on-white-150px.png</Logo>
```

Comment:
I understand that this file has to do with edge webstore, do we know if there is any proper way to test it locally?
@peterdur may have some ideas.

Reply:
@Shobhit1 can you follow up with @peterdur and see if there is anything we may want to test further before closing this PR?